tests/test_plotter.py

```diff
@@ -253,3 +253,15 @@ def test_add_multiple(sphere):
     pl.add_mesh(sphere, scalars='data')
     pl.show()
     assert sphere.n_arrays == 1
+
+
+def test_deep_clean(cube):
+    pl = pyvista.Plotter()
+    cube_orig = cube.copy()
+    pl.add_mesh(cube)
+    pl.deep_clean()
+    assert pl.mesh is None
+    assert pl.mapper is None
+    assert pl.volume is None
+    assert pl.textActor is None
+    assert cube == cube_orig
```

Comment:
This might be a bit paranoid, but I've added a regression test to make sure we're not messing with the mesh. I changed the mesh fixture from the `sphere` to the `cube` because the latter has some point/cell data.

Reply:
Good point. I agree with the changes.